### PR TITLE
Update text Site-Layout-Files _include file

### PR DESCRIPTION
**Pull request by mjones01**
@lwasser Please review the new text and accept/ suggest edits as necessary.  

The previous text for the button was limited to Harvard Forest.  I updated the description text to remove the flux tower specifics as the shapefiles aren't directly tied to it.  I also added text for the Census origin of the other shapefiles.  

Two questions: 
Are the HARV forest shapefiles from NEON or do we need to add a link to another site?  
Does the 30 years/proxy data bit seem weird in this shapefile context? (direct copy from Met-Time-Series). 

The download address is also updated to the current files.  

I'm updating this now as I need it for a screen shot in the SetWD lesson.

### CODE REVIEW

File: _includes/dataSubsets/_data_Site-Layout-Files.html [hunk 1]
```diff
@@ -1,9 +1,12 @@
-<div><a href="https://ndownloader.figshare.com/files/3634374" class="btn btn-success">
-Download NEON Teaching Data Subset: Harvard Forest Shapefiles</a></div>
+<div><a href="https://ndownloader.figshare.com/files/3672549" class="btn btn-success">
+Download NEON Teaching Data Subset: Location Boundary Shapefiles</a>
+</div>
```
Comment (mjones01) [inline _includes/dataSubsets/_data_Site-Layout-Files.html]: Now I'm thinking sticking with the file name for text here: 
Download NEON Teaching Data Subset: Site Layout Shapefiles

Will make change prior to PR merge, if you agree.

Comment (lwasser) [inline _includes/dataSubsets/_data_Site-Layout-Files.html]: I don't have a strong opinion either way.  Make the call!

**Comment by lwasser** [comment]
The shapefiles are MOSTLY from the Harvard forest site. However the plot locations are NEON data.

File: _includes/dataSubsets/_data_Site-Layout-Files.html [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
 
 The vector data used in this lesson were collected at the 
 <a href="http://www.neoninc.org/" target="_blank"> National Ecological Observatory Network's</a> 
-<a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank"> Harvard Forest field site.</a> 
-These data are proxy data for what will be available for 30 years from the NEON
-flux towers from the 
-<a href="http://data.neoninc.org/" target="_blank"> NEON data portal</a>.
\ No newline at end of file
+<a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank"> Harvard Forest field site</a>.
+These data are proxy data for what will be available for 30 years on the
```
Comment (lwasser) [inline _includes/dataSubsets/_data_Site-Layout-Files.html]: yes maybe remove the proxy data sentence for now. they aren't NEON data at all really. Just site characterization / infrastructure data.

Then add a link to the harvard forest GIS page - Thanks!

**Comment by mjones01** [comment]
@lwasser _includes with links to clean .zips now ready for review/ merge.  

Once accepted we need to copy to the other repos (Raster, Vector, Time-Series, and Lesson Development all have lessons using these files).

**Comment by lwasser** [comment]
awesome. thank you! I'm going to copy them to vector now. Feel free to copy to raster and time series.

**Pull request merged**